### ISSUE TITLE
Remplacer isHeader par drop

### ISSUE THREAD

**Issue by Guigo91** [lraulier/afterlunch-draft#1]
Je pense que retirer la première ligne serait plus efficace que de regarder toutes les lignes une à une.
La fonction drop(nbLines) devrait faire l'affaire.

**Comment by lraulier** [comment]
oui, la dédection du header est toute pourrie .. faut la modifier 
je vais tester drop asap :)

**Comment by lraulier** [comment]
la méthode n'est pas implémentée il me semble pour un RDD
sparkContext.textFile("../data/sample.csv").drop(1) n'est donc pas possible!
De quel façon penses tu utiliser la méthode drop ???

**Comment by lraulier** [comment]
http://erikerlandson.github.io/blog/2014/07/27/some-implications-of-supporting-the-scala-drop-method-for-spark-rdds/

**Comment by lraulier** [comment]
ps : en passant par spark sql , le résultat est immédiat 
cf branch useDataFrame

**Comment by Guigo91** [comment]
J'avais vu pour useDataFrame, dommage qu'on ne puisse pas faire ça facilement en Spark.
Là il y a un gars qui a implémenter le fonction drop : 
https://github.com/erikerlandson/spark/commit/7953b9bf25ab4183b726150576e4f334a1cb2694
Ca devient tout de suite plus compliqué quand on manipule des partitions.

**Comment by lraulier** [comment]
j'arrive pas a savoir si cette method a été commitée dans le project spark.
je la trouve sur redhat silex ... mais pas sur qu'elle soit native ...
@suivre

**Comment by lraulier** [comment]
demande de merge : https://github.com/apache/spark/pull/1839

**Comment by Guigo91** [comment]
J'imagine que faire la même fonction que lui mais en ne supprimant que la première ligne de la première partition, c'est un peu laborieux pour juste retirer un header...
